F-Droid can't build

```
...
> Task :app:processProductReleaseMainManifest FAILED
> Task :app:mergeProductReleaseResources

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processProductReleaseMainManifest'.
> com.android.manifmerger.ManifestMerger2$MergeFailureException: Error parsing /home/vagrant/build/net.foucry.pilldroid/app/src/main/AndroidManifest.xml
``` 

for https://github.com/jfoucry/Pilldroid/releases/tag/v0.350-beta

/LE: fyi https://gitlab.com/fdroid/fdroiddata/-/commit/fd3173562e77e4563496d0b96b1b9116b0813ae8

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:tools="http://schemas.android.com/tools">
 
 
     <uses-permission android:name="android.permission.CAMERA" />

